app/controllers/miq_ae_tools_controller.rb

```diff
@@ -333,6 +333,42 @@ def reset_datastore
     javascript_flash(:spinner_off => true)
   end
 
+  def get_form_vars_react
```

Comment:
I think its best to change the name resolve_react to a different name which aligns with our feature.